docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -419,6 +419,10 @@ NB: You must `REDIRECT_MACARON_LOG` and have `DISABLE_ROUTER_LOG` set to `false`
 - `RUN_AT_START`: **true**: Run repository statistics check at start time.
 - `SCHEDULE`: **@every 24h**: Cron syntax for scheduling repository statistics check.
 
+### Cron - Update Migration Poster ID (`cron.update_migration_post_id`)
+
+- `SCHEDULE`: **@every 10m** : Interval as a duration between each synchronization, it will always be started at start.
```

Comment:
```suggestion
- `SCHEDULE`: **@every 10m** : Interval as a duration between synchronization attempts, the first attempt will be at the time of the migration.
```
Or perhaps I don't understand what "at start" means in this context?

Reply:
No. it means when gitea start running.

Reply:
When migration, the posterid will be replaced directly.

Reply:
OK, then I think this is more clear:

```suggestion
- `SCHEDULE`: **@every 10m** : Interval as a duration between each synchronization, it will always attempt synchronization when the instance starts.
```